### PR TITLE
create base for mail template

### PR DESCRIPTION
**Pull request by Hardevv**
DONE:
services/mailer/mailTemplate.ts | should be refactorized, should return an object of all templates
static/mailTemplates/cryptoAlert.ts | function that returns HTML and attachments should have some parameters like cryptocurrency name, price and alert text to be displayed inside the mail
list of icons mapped into email 
refactorize
TODO:

### CODE REVIEW

**Comment by haron-IV** [comment]
test

**Comment by haron-IV** [comment]
test

**Comment by haron-IV** [comment]
halo

**Comment by haron-IV** [comment]
test

**Comment by haron-IV** [comment]
ahha

File: src/app/app.ts [hunk 1]
```diff
@@ -31,7 +32,7 @@ const priceIsUnderExpectation: PriceCheckFn = (
   convertedTo
 ) => {
   const mail = priceIsUnder(cryptocurrencyName, alertPrice.under, convertedTo)
-  mailer.sendMail(mail.subject, mail.html)
+  mailer.sendMail(mail.html as string, mail.subject, mail.attachments)
```
Comment (Hardevv) [inline src/app/app.ts]: `mailer.sendMail()` should takes one parameter `object`

File: src/services/Mailer/Mailer.ts [hunk 2]
```diff
@@ -15,10 +16,15 @@ class Mailer {
     },
   })
 
-  sendMail = (text: string, subject: string) => {
+  sendMail = (
+    html: string | ReadStream,
+    subject: string,
+    attachments?: any[]
```
Comment (Hardevv) [inline src/services/Mailer/Mailer.ts]: it should be generic typed

File: src/services/Mailer/mailTemplate.ts [hunk 3]
```diff
@@ -1,39 +1,45 @@
 import { SupportedCryptocurrencies } from 'app'
+import { ReadStream } from 'fs'
+import { template } from '@root/static/mailTemplates/cryptoAlert/template'
```
Comment (haron-IV) [inline src/services/Mailer/mailTemplate.ts:3]: instead of using `@root` use `@static` path

**Pull request merged**